Doc/library/tk.rst

```diff
@@ -1,7 +1,7 @@
 .. _tkinter:
 
 *********************************
-Graphical User Interfaces with Tk
+Graphical user interfaces with Tk
```

Comment:
Graphical User Interfaces is commonly initialised as GUI, so I think caps here are reasonable:
```suggestion
Graphical User Interfaces with Tk
```